app/views/pages/developer-guide/records/crud-records-using-ajax.liquid

```diff
@@ -156,7 +156,7 @@ Then you need to create a page that returns JSON with the data:
 ```liquid
 {%- raw -%}
 {%- graphql recent = "modules/feedback/feedback", per_page: 10 -%}
-{{ recent.models.results | json }}
+{{ recent.records.results | json }}
```

Comment:
no need for `| json` filter, `{{ recent.records.results }}` will give the same results ; `| json` was necessary loong time ago. I know it's unrelated, just noting it here.

Reply:
I opened a ticket for this because there might be other examples that include this filter: #1337